web-app/src/api/restApi.js

```diff
@@ -143,6 +143,11 @@ const api = {
             }, {...config, path: '/reports/create'})
         },
     },
+    nodes: {
+        getDefaultNode: () => fetch('http://thisnode.info/cgi-bin/hostname')
```

Comment:
In my current network this fetch doesn't fail (and triggers catch), but hangs for ever. I think we can add a timeout here so when the user isn't connected to the mesh we can show a friendly message.

Reply:
Does fetch have a timeout?

Reply:
no, I think we should add our own timeout

Reply:
You can try now?